src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    private static final int MIN_PASSWORD_LENGTH = 4;
+    @NotNull
+    @Email
```

Comment:
you have custom validation, why do you still use `@NotNull` here?

Reply:
donw